app/src/main/java/com/codepath/apps/critterfinder/activities/PetBrowserActivity.java

```diff
@@ -107,6 +111,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data) {
         }
     }
 
+    @Override
+    public void onPetSelected(PetModel pet, View transitionView) {
+        ActivityOptionsCompat options = ActivityOptionsCompat.makeSceneTransitionAnimation(this, transitionView, "details");
+        Intent intent = PetDetailsActivity.getStartIntent(this, pet);
+        intent.setFlags(Intent.FLAG_ACTIVITY_SINGLE_TOP);
+        startActivity(intent, options.toBundle());
+    }
```

Comment:
use a scene transition to go from the card view to the details view with the pet image as the  transition view